tck/src/main/resources/tests/startPatrolling/input.graphql

```diff
@@ -0,0 +1,6 @@
+mutation heroStartPatrolling {
```

Comment:
Needs cleanup - and maybe use a different time to differentiate between other tests?